src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/equalshashcode/Example1.java

```diff
@@ -0,0 +1,76 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="EqualsHashCode"/>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.equalshashcode;
+
+// xdoc section -- start
+public class Example1 {
+  public int hashCode() { // violation, no valid 'equals'
+    // code
+    return 0;
+  }
+  public boolean equals(String o) {
+    // code
+    return false;
+  }
+}
+class ExampleNoHashCode {
+  public boolean equals(Object o) { // violation, no 'hashCode'
+    // code
+    return false;
+  }
+  public boolean equals(String o) {
+    // code
+    return false;
+  }
```

Comment:
@Lmh-java ,
Extremely  sorry for the previous post. Please ignore it. I completely thought the files here were xpath as u were previously working on that issue.
We cannot split the example files here because these pages are our docs which the end users refer to.

You can instead do this : 

- I think it is okay to remove complete  `// code` lines to comply with the length
- Remove blank line no.8 and no.10

If this still exceeds, then we don't have any other option than to revert and suppress the check for this particular file.
Again, sorry for the previous review

Reply:
No worries, I will try to compress as what you suggest here 👍. Thanks